iFixit/react-commerce
# Mini-Cart: Fix Empty Cart Icon

## Overview

The Empty Cart icon was just a crying emoji, and so we wanted to replace that with the design [here](https://www.figma.com/file/sfj6r7Bn6eREMvuqo0f7Du/iFixit---Components?node-id=284%3A17687). We also wanted to make sure that the cart icon works on on Windows 10 and lower on Chrome (102,103, and 104-beta).

This adds `Font Awesome` dependencies to use the specified design. 

## QA
Does the install work fine? You will have to add the `FONT_AWESOME_NPM_TOKEN` to the proper `.env` file for this install to work.

Click on the cart icon on your app and make sure you don't have any items added. If it doesn't the previously mentioned design should now show up.

Closes: #412
